source/includes/_aviate-health-apis.md

```diff
@@ -372,7 +371,18 @@ curl -X GET \
 
 > Example Response:
 
-TODO
+````json
+[
+  {
+    "nodeName": "",
```

Comment:
I can't see in this PR the 'model' being defined, but regardless we need to explain what these fields are, and in particular since we expose `eventGroup` and `sampleKind` explain what those are and their values (e.g.  sampleKind={gauge, meter, timer}). This can happen in a subsequent PR.

Reply:
Models are already documented in the doc. See [this section](https://killbill.github.io/slate/aviate-health-apis.html#models). We have a [SamplesForSampleKindAndHost](https://killbill.github.io/slate/aviate-health-apis.html#samplesforsamplekindandhost) model. But yes, the fields in this model need to be explained in detail (which is why I had filed [#185](https://github.com/killbill/technical-support/issues/185) in the first place). We should also rename this to `MetricResult` as discussed on Slack.

Reply:
I will address this is a separate PR.

Reply:
Opened https://github.com/killbill/slate/pull/329